src/scripts/main.js

```diff
@@ -1,7 +1,26 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.querySelector('body');
+  const element = document.createElement('div');
+  const heading = document.createElement('h2');
+  const message = document.createElement('p');
+
+  body.append(element);
+
+  element.style.top = posTop + 'px';
+  element.style.right = posRight + 'px';
+
+  element.className = `notification + ${type}`;
```

Comment:
```suggestion
  element.className = `notification  ${type}`;
```
![Screenshot from 2023-03-21 21-37-07](https://user-images.githubusercontent.com/33029864/226721951-bc73bff4-80b7-4a76-8112-7b03fa088ffc.png)